<ui version="4.0" >
 <class>SetupQslCard</class>
 <widget class="QDialog" name="SetupQslCard" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Setup QSL Card Print</string>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="4" row="0" column="0" >
    <widget class="QTableView" name="qsoFields" >
     <property name="maximumSize" >
      <size>
       <width>150</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="4" row="0" column="2" >
    <widget class="QTableWidget" name="selectedFields" >
     <column>
      <property name="text" >
       <string>Entry</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>X Position ( mm )</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Y Position ( mm )</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Condition</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Value</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Font Size</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="add" >
     <property name="maximumSize" >
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text" >
      <string>></string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QPushButton" name="remove" >
     <property name="maximumSize" >
      <size>
       <width>20</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text" >
      <string>&lt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item rowspan="2" row="4" column="2" >
    <widget class="QTableWidget" name="extraValues" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <column>
      <property name="text" >
       <string>Text</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>X Position ( mm )</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Y Position ( mm )</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Font Size</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="6" column="1" colspan="2" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QPushButton" name="addExtraText" >
     <property name="text" >
      <string>Add additional Print Text</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <widget class="QPushButton" name="removeExtra" >
     <property name="text" >
      <string>Remove additional Text</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SetupQslCard</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetupQslCard</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
